src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpWindowsProxy.cs

```diff
@@ -178,6 +206,8 @@ public void Dispose()
                 {
                     SafeWinHttpHandle.DisposeAndClearHandle(ref _sessionHandle);
                 }
+
+                _cacheCleanupTimer.Dispose();
```

Comment:
Is it possible '_cacheCleanupTimer' might be null in some error cases where the initialization code threw?  If so, this should be '_cacheCleanupTime?.Dispose()'

Reply:
No; it's created in ctor and intentionally the last thing that gets initialized to avoid a stray timer.

Realistically, `Dispose()` will never actually be called as this class is instantiated as a singleton.